pkg/apis/serving/v1alpha1/domainmapping_lifecycle.go

```diff
@@ -62,7 +62,7 @@ const (
 	// AutoTLSNotEnabledMessage is the message which is set on the
 	// DomainMappingConditionCertificateProvisioned condition when it is set to True
 	// because AutoTLS was not enabled.
-	AutoTLSNotEnabledMessage = "autoTLS is not enabled"
+	AutoTLSNotEnabledMessage = "AutoTLS is not enabled"
```

Comment:
should we update the corresponding message in [route_lifecyce.go](https://github.com/knative/serving/blob/3695f5b03a9cc4dc2ef83741e0481f4c95e98094/pkg/apis/serving/v1/route_lifecycle.go#L174) to match?

Reply:
I will follow up by making both changes in the same PR. for now I reverted this line.